itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/launch/configuration/AbstractLaunchConfigurationTest.java

```diff
@@ -39,16 +36,16 @@
 
 public abstract class AbstractLaunchConfigurationTest {
 	public static void createNewQuarkusProject(String projectName, String projectType) {
-		AbstractCreateNewProjectTest.testCreateNewProject(projectName, projectType);
-		AbstractCreateNewProjectTest.checkJdkVersion(projectName, projectType);
+		AbstractNewQuarkusProjectTest.testCreateNewProject(projectName, projectType);
+		AbstractNewQuarkusProjectTest.checkJdkVersion(projectName, projectType);
 
-		AbstractCreateNewProjectTest.checkProblemsView();
+		AbstractNewQuarkusProjectTest.checkProblemsView();
```

Comment:
This is also in several files. These three methods could be also in AbstractTest class or some util class for creating project.